docs/en/serverless/logging/monitor-datasets.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: serverlessObservabilityMonitorDatasets
+slug: /serverless/observability/monitor-datasets
+title: Monitor log data set quality
+description: Monitor log data sets to find degraded documents.
+tags: [ 'serverless', 'observability', 'how-to' ]
+---
+
+<p><DocBadge template="technical preview" /></p>
+<p><DocBadge template="beta" /></p>
+
+The **Data Set Quality** page provides an overview of your log data sets.
+Use this information to get an idea of your overall log data set quality and find data sets that contain incorrectly parsed documents.
+Access the Datasets page from the main ((kib)) menu and go to **Stack Management** → **Data Set Quality*.
+
+{/* Update screenshot when changes are done <DocImage size="2" url="../images/logs-dataset-overview.png" alt="Screen capture of the data set overview" /> */}
+
+<DocCallOut title="Requirements">
+  Users with the `viewer` role can see the Datasets Quality summary. To see the Active Datasets and Estimated Data summaries, users need the `monitor` [index privilege](((ref))/security-privileges.html#privileges-list-indices) for the `logs-*-*` index.
```

Comment:
Should this be `Data sets Quality summary`?
<img width="952" alt="image" src="https://github.com/elastic/observability-docs/assets/1313018/13e45ccc-e1a3-42cb-af39-f417d676f040">